cisco_asa/plugin.spec.yaml

```diff
@@ -189,3 +189,37 @@ actions:
         description: Success if address add to group
         type: boolean
         required: true
+  create_address_object:
+    title: Create Address Object
+    description: Create address object by the object IP address
+    input:
+      address:
+        title: Address
+        description: IP, CIDR, or domain name to assign to the address object
```

Comment:
This says domain and CIDR is supported when the action description on line 194 says "IP address"

We need to update one of these lines to be correct and match the other.

Reply:
@jschipp-r7  I changed this description in both places. Action allow to create address object with ipv4, ipv6, fqdn, cidr and range.